src/MessagingInput/MessagingInput.tsx

```diff
@@ -173,8 +174,8 @@ const MessagingInputRenderFunction: React.ForwardRefRenderFunction<MessagingInpu
               <span className={cssClass("SendText")}>{sendButtonText}</span>
             </>
           }
-          // Disable the Send if nothing but whitespace is in the input field. Also disable if it is manually disabled
-          disabled={!value.trim() || disableSendButton}
+          // Disable Send if body is only whitespace and there are no attachments, or if manually disabled
+          disabled={(value.trim() === "" && !attachments?.length) || disableSendButton}
```

Comment:
This is a-okay, but - even though it's long - I think `(!attachments || attachments.length === 0)` for the second condition might be a bit easier to read.

Annoyingly, I think this might be a good time to extract this line out into its own function, an `isSendButtonDisabled` helper or such. 😄

Reply:
Addressed this and similar previous comment in https://github.com/Clever/components/pull/681/commits/ae87a6964750db151d90936bc0643f0d5a43072e